targets/targets.json

```diff
@@ -7642,6 +8071,8 @@
             "PSOC6"
         ],
         "device_name": "CY8C6347BZI-BLD53",
+        "mbed_rom_start": "0x10000000",
+        "mbed_rom_size": "0x100000",
         "mbed_ram_start": "0x08002000",
         "mbed_ram_size": "0x00045800",
```

Comment:
Are these RAM values correct? They seem to have similar issues to the `CY8C624ABZI-D44` issues.